modules/policy/engine-testing/src/main/java/org/eclipse/sw360/antenna/policy/testing/GivenSteps.java

```diff
@@ -54,6 +56,11 @@ private void mapArtifactFacts(List<List<String>> artifact_facts, Artifact artifa
         artifact_facts.forEach(row -> mapProperty(row, artifact));
     }
 
+    private static final String LICENSES_PROP = "licenses";
```

Comment:
In cases like this one, where the constants are connected to a specific part in the code, I prefer to have them defined close to the usage because it helps reading the code. If there are multiple groups of constants all defined at the top of a file, it gets awkward to relate them to to each other and to their meaning.